app/lib/three_scale/invitation_email_delivery_observer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ThreeScale::InvitationEmailDeliveryObserver
+  def self.delivered_email(message)
+    binding.pry
+    Invitation.find_by(email: message.to)
+              .update_column(:sent_at, Time.zone.now)
```

Comment:
If this works, this is better because it does not skip validations:
```suggestion
              .update!(sent_at: Time.zone.now)
```